lib/chef/knife/vsphere_vm_clone.rb

```diff
@@ -566,14 +605,18 @@ def generate_clone_spec(src_config)
 
     if config[:customization_ips]
       if get_config(:customization_gw)
-        cust_spec.nicSettingMap = config[:customization_ips].split(',').map { |i| generate_adapter_map(i, get_config(:customization_gw)) }
+        cust_spec.nicSettingMap = config[:customization_ips].split(',').map.with_index { |i, index|
```

Comment:
I'd defer to the person who used i originally--I only added "index" to that list due to switching to using with_index.  It appears that it represents one of the ips from config[:customization_ips] so 'ip' is probably a decent name for it, if that's not used already.